text_extensions_for_pandas/io/test_tokenization.py

```diff
@@ -17,15 +17,33 @@
 
 import pandas as pd
 import numpy as np
-from transformers import BertTokenizerFast
+import textwrap
+from typing import *
+
+# noinspection PyPackageRequirements
+from transformers import BertTokenizerFast, BertModel
 
 from text_extensions_for_pandas.io.tokenization import (
-    make_bert_tokens, windows_to_seq, seq_to_windows
+    make_bert_tokens, windows_to_seq, seq_to_windows, add_embeddings, conll_to_bert
+)
+from text_extensions_for_pandas.io.conll import (
+    conll_2003_to_dataframes, make_iob_tag_categories
+)
+from text_extensions_for_pandas.array import (
+    TensorArray
 )
 
 
 class TestTokenize(unittest.TestCase):
 
+    def __init__(self, *args, **kwargs):
```

Comment:
maybe better to put this in `setupClass(cls)`?

Reply:
Very good idea. I will make that change. I have some additional changes to that file already in my local copy, so I'll merge the current set of changes and include the `setupClass()` method in the next batch.